src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -651,6 +656,8 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Shutdown()
     if (rejit_handler != nullptr)
     {
         rejit_handler->Shutdown();
+        delete rejit_handler;
```

Comment:
out of curiosity, why not a `std::unique_ptr` ?

Reply:
because I'm interested in releasing all resources before the CorProfiler instance dtor... I know I can do it by calling a method on `std::unique_ptr` too but I leave it like this and leave the `std::unique_ptr` to the scenario where it follows the complete lifecycle of the parent.

Reply:
sure no pb, just curiosity ;)